src/http2_connection.erl

```diff
@@ -1378,54 +1351,15 @@ socksend(#connection{
             {error, Reason}
     end.
 
-%% Stream list operations. Warning: These are abusing the underlying
-%% tuple structure of the record for efficiency's sake
-
--spec get_stream(stream_id(), [stream()]) -> stream() | false.
```

Comment:
This stuff used to be in http2_connection, but I moved these functions into h2_streams to make it easier to follow, then rewrote them to use the streams data structure.